src/core/edgeNetwork/injectApplyAepEdgeResponse.js

```diff
@@ -0,0 +1,35 @@
+import { createCallbackAggregator, noop } from "../../utils";
+import mergeLifecycleResponses from "./mergeLifecycleResponses";
+
+export default ({ cookieTransfer, lifecycle, createResponse }) => {
+  return ({
+    request,
+    responseHeaders,
+    responseBody,
+    runOnResponseCallbacks = noop
+  }) => {
+    const onResponseCallbackAggregator = createCallbackAggregator();
+    onResponseCallbackAggregator.add(lifecycle.onResponse);
+    onResponseCallbackAggregator.add(runOnResponseCallbacks);
+
+    return lifecycle
+      .onBeforeRequest({
+        request,
+        onResponse: onResponseCallbackAggregator.add,
+        onRequestFailure: noop
+      })
+      .then(() => {
```

Comment:
Now that the customer is including the entire response, It may be useful to call processWarningsAndErrors like we do in sendEdgeNetworkRequest. This would catch a case where experience edge returns an error, but the customer doesn't know how to handle it. This also opens up the possibility that the "call" could fail in which case we should call the failure callbacks.